tools/run_tests/artifacts/build_package_python.sh

```diff
@@ -13,15 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex
+set -eux
 
 cd "$(dirname "$0")/../../.."
 
 mkdir -p artifacts/
 
 # All the python packages have been built in the artifact phase already
 # and we only collect them here to deliver them to the distribtest phase.
-cp -r "${EXTERNAL_GIT_ROOT}"/input_artifacts/python_*/* artifacts/ || true
+find "${EXTERNAL_GIT_ROOT}"/input_artifacts/ \
+    -maxdepth 1 \
+    -type d \
+    -name "${ARTIFACT_PREFIX}*" \
+    -not -name "${EXCLUDE_PREFIX}" \
+    -print0 \
+        | find -files0-from - -type f -maxdepth 1 -exec 'cp -v {} ./artifacts || true' \;
```

Comment:
```suggestion
        | xargs -0 -I% find % -type f -maxdepth 1 -exec cp -v {} ./artifacts \;
```

Since `-files0-from -` isn't supported by the older versions of `find`.